### PR TITLE
Fix cursor Bug

### PR DESCRIPTION
**Pull request by sajjaadf**
Just fix issue #2902

### CODE REVIEW

**Comment by uberhacker** [comment]
Thank you for the PR @sajjaadf!  Please see the comment.

**Comment by JL2210** [comment]
Are those Mac line endings?

**Comment by sajjaadf** [comment]
which comment?

File: packages/scite.rb [hunk 1]
```diff
@@ -25,6 +25,8 @@ class Scite < Package
   depends_on 'hicolor_icon_theme'
   depends_on 'shared_mime_info'
   depends_on 'sommelier'
+  depends_on 'libxcursor' 
+  depends_on 'xcb_util_cursor'
```
Comment (uberhacker) [inline packages/scite.rb]: Maybe these should be included in the sommelier package to other apps don't have cursor issues?  In any event, we should be adding new dependencies above sommelier so the postinstall messages don't scroll past the users' visible screen area.

Comment (sajjaadf) [reply packages/scite.rb]: okay

**Pull request closed without merging**